test/testmachinery/shoot/enable_disable_test.go

```diff
@@ -26,6 +27,8 @@ const (
 var _ = Describe("Shoot registry cache testing", func() {
 	f := framework.NewShootFramework(nil)
 
+	isVerticalPodAutoscalerDisabled := false
```

Comment:
Cosmetic nits:
- Here `isVerticalPodAutoscalerDisabled := false` is used, the other 2 tests use `var isVerticalPodAutoscalerDisabled bool`.
- We could group all vars in a dedicated var block:
```golang
	var (
		f = framework.NewShootFramework(nil)

		isVerticalPodAutoscalerDisabled bool
	)
```